python/ray/serve/tests/test_handle.py

```diff
@@ -186,16 +186,21 @@ def __call__(self):
             return "__call__"
 
     handle1 = serve.run(MultiMethod.bind())
-    metrics = handle1.request_counter
+    counter = handle1.request_counter
+    counter_info = counter.info
     assert handle1.remote().result() == "__call__"
 
     handle2 = handle1.options(method_name="method_a")
+
     assert handle2.remote().result() == "method_a"
-    assert handle2.request_counter == metrics
+    assert handle2.request_counter == counter
+    assert handle2.request_counter.info == counter_info
```

Comment:
This should catch default tags being overridden previously

Reply:
Thanks for adding the test!